src/python_interpreter/mod.rs

```diff
@@ -247,16 +265,10 @@ fn find_all_windows(
     Ok(interpreter)
 }
 
-struct WindowsPythonInfo {
-    major: usize,
-    minor: usize,
-    pointer_width: Option<usize>,
-}
-
 fn windows_python_info(executable: &Path) -> Result<Option<WindowsPythonInfo>> {
     let python_info = Command::new(executable)
         .arg("-c")
-        .arg("import sys; print(sys.version)")
+        .arg("import sys, sysconfig; print(sysconfig.get_platform(), sys.version_info.major, sys.version_info.minor)")
```

Comment:
If you accept my suggestion that platform might be best at the end

```suggestion
        .arg("import sys, sysconfig; print(sys.version_info.major, sys.version_info.minor, sysconfig.get_platform())")
```